### PR TITLE
Fix credit card number format.

### PR DESCRIPTION
**Pull request by nuxzero**
This PR fixed issue #27. Because of `OnKeyListener` will fired only hardware keyboard and not working with alternative keyboards. To solve format on `CreditCardEditText`, use `TextWatcher` instead of `OnKeyListener` to format credit card number.

### CODE REVIEW

**Comment by nuxzero** [comment]
@pitiphong-p @chakrit please review krub.

File: sdk/build.gradle [hunk 1]
```diff
@@ -80,6 +81,13 @@ uploadArchives {
 dependencies {
     compile 'joda-time:joda-time:2.9.2'
     compile 'com.squareup.okhttp3:okhttp:3.4.1'
+    compile 'com.android.support:appcompat-v7:26.1.0'
```
Comment (pitiphong-p) [inline sdk/build.gradle]: Can we do this without support library? We try to avoid that for the compatibility issue with our client.

Comment (nuxzero) [reply sdk/build.gradle]: This one using for testing with `AppCompatActivity`. Btw I can change to normal `Activity` and delete support library.

Comment (nuxzero) [reply sdk/build.gradle]: @pitiphong-p please review gain krub.

**Pull request merged**